compiler/src/iree/compiler/Codegen/Common/GPU/DecomposeHorizontallyFusedGemms.cpp

```diff
@@ -45,7 +45,9 @@ static LogicalResult captureUsedOperationsAndBlockArguements(
   auto yieldOp = cast<linalg::YieldOp>(linalgOp.getBlock()->getTerminator());
   Value result = yieldOp.getOperand(resultNumber);
 
-  getBackwardSlice(result, &usedOperations, options);
+  [[maybe_unused]] LogicalResult ret =
```

Comment:
nit: llvm had a discussion on using `maybe_unsuded` vs. casting to void and decided on sticking to the latter: https://discourse.llvm.org/t/maybe-unused-vs-attribute-unused-vs-void-var-for-assert-variables/64212

Reply:
Thanks, I did not know this! I don't really follow why the conclusion is sticking to the latter. It seems that there is not enough data to say which option wins? I don't have a preference, and I'm okay with both. The reason that I use `maybe_unused` is just because it looks more modern and descriptive. It looks friendly to people who don't use c++ as their daily programming language, IMO.